projects/plugins/crm/composer.json

```
@@ -3,7 +3,7 @@
 	"description": "[Jetpack CRM](https://jetpackcrm.com) Core CRM Plugin - WordPress.org Hosted",
 	"require-dev": {
 		"automattic/jetpack-changelogger": "@dev",
-		"codeception/codeception": "^4.1 || ^5.0",
+		"codeception/codeception": "^4.1 || ^5.3",
```

We may as well leave this as it was. 8.2+ will get 5.3 anyway, while 8.0 can use 5.1.2 instead of 4.2.2 (and if they fix 8.1, that can use 5.2.2 instead of 4.2.2).
```suggestion
		"codeception/codeception": "^4.1 || ^5.0",
```